### DISCUSSION TITLE
Documentation rework ideas

### DISCUSSION THREAD

**Discussion by rbrian** [Ralim/IronOS#1263] [Ideas]
- ~~For starters, I'm going to edit the Wiki to replace all the updating/firmware info with links to the relevant pages on the github.~~ done 
- @River-b will start a discussion topic to hold the most recent charger/cable/power supply list, wiki will be adjusted to link to it 
- Work with @CRImier to implement his design ideas

Other documentation might be massaged along the way. Suggestions/comments are welcome.

**Comment by River-Mochi** [comment]
might want to talk to @Crimier  , has a a design set up for wiki to help guide people to the Pinecil Chat for troubleshooting and help but has  not  had time to implement it.  he has screen shots of design layout he PM me that were nice and better organized than the current Pine64 wiki.  maybe if you help him implement it could get done.

**Reply by rbrian** [reply]
will do. much of the existing page is good info, but often painful to read for one reason or another. :)

**Reply by rbrian** [reply]
he's been pinged, will discuss when he's off work
